test/libsolidity/semanticTests/immutable/getter_call_in_constructor.sol

```diff
@@ -0,0 +1,15 @@
+contract A {
+    uint immutable public x = 1;
+    uint public y;
+    constructor() public {
+        y = this.x();
```

Comment:
This will warn about the use of ``this`` in the constructor, right? And we can't really make it an error, since then we'd also need to track ``A a = this; a.x();``, etc., and that'd become impossible quickly, right?

Reply:
exactly